src/python/pants/scm/subsystems/changed.py

```diff
@@ -33,6 +35,7 @@ class DependeesOption(Enum):
 class ChangedRequest:
```

Comment:
It's a little confusing how this is different from `ChangedFilesRequest`. Would you mind please renaming this to `ChangedAddresessRequest`?